crates/bevy_render/src/render_asset.rs

```diff
@@ -25,13 +25,16 @@ pub enum PrepareAssetError<E: Send + Sync + 'static> {
 pub trait RenderAsset: Asset {
     /// The representation of the asset in the "render world".
     type ExtractedAsset: Send + Sync + 'static;
+    /// Specifies all ECS data required by [`RenderAsset::extract_asset`].
+    /// For convenience use the [`lifetimeless`](bevy_ecs::system::lifetimeless) [`SystemParam`].
```

Comment:
It would be nice to have a doc test or two demonstrating how this might be used here.

In particular, fetching multiple data types via tuples or the derive macro will be unintuitive to new users.

Reply:
This pattern is copied from `RenderAsset::prepare_asset`, I think users can refer to it.